vite.config.ts

```diff
@@ -40,6 +40,8 @@ const config: UserConfig = {
   define: {
     // eslint-disable-next-line @typescript-eslint/naming-convention
     "process.env": { READABLE_STREAM: "disable" },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    HASH_ROUTING: process.env.HASH_ROUTING !== undefined,
```

Comment:
```suggestion
    HASH_ROUTING: Boolean(process.env.HASH_ROUTING),
```

Reply:
I wonder if we should just set it inside `process.env` on the line above :thinking:

Reply:
Done with 79463006